### DISCUSSION TITLE
[Import fitlogex] When importing a fitlogex, what deetermines which MTB tag is assigned?

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#967] [General]
I find that if I replace a piece of equipment (e.g. derailleur cable) in SportTracks and later export a fitlogex of the tour which used the new equipment than the MTB tag uses a prior item with the same Tag name but different ST Id# and other different field data (e.g. purchase date).

Example from a fitlogex and MTB in screen shot.
I think this means that each piece of equipment has to have a unique Tag name to be a useful tag.

![MTB tag issues](https://user-images.githubusercontent.com/106929478/203441782-1d03857f-d580-499b-acad-7caf26f914d6.jpg)

**Comment by FJBDev** [comment]
>When importing a fitlogex, what deetermines which MTB tag is assigned?

Looking at [the code](https://github.com/mytourbook/mytourbook/blob/a538b7874d913ad57bd6456e316d4e2bb73d21b1/bundles/net.tourbook/src/net/tourbook/importdata/RawDataManager.java#L786), it seems that the name AND the tag id (in the note) is what it's used to recognize a tag as duplicate vs a new one.
Hence, in your case, a new tag for "Rear Derailleur" should be created.
Can you create a bug request with a scenario on how to reproduce the issue ?

Thanks
